src/test/dumps/59c86272-03ea-42e9-a62d-1c8a272e8ab0

```diff
@@ -0,0 +1,130 @@
+["createOfferOnSuccess","PC_0","",1651144298809]
+["setLocalDescription","PC_0","",1651144298811]
+["onconnectionstatechange","PC_0","connected",1651144298813]
+["onnegotiationneeded","PC_1",null,1651144298815]
+["onsignalingstatechange","PC_0","have-local-offer",1651144298818]
+["setLocalDescriptionOnSuccess","PC_0",null,1651144298818]
+["setRemoteDescription","PC_0","",1651144298821]
+["setRemoteDescription","PC_1","",1651144298839]
+["onsignalingstatechange","PC_0","stable",1651144298840]
+["setRemoteDescriptionOnSuccess","PC_0",null,1651144298840]
+["setRemoteDescription","PC_0","",1651144298846]
+["onsignalingstatechange","PC_1","have-remote-offer",1651144298848]
+["ontrack","PC_1","audio:mixedlabelaudio0 stream:mixedmslabel",1651144298850]
+["ontrack","PC_1","video:mixedlabelvideo0 stream:mixedmslabel",1651144298850]
+["setRemoteDescriptionOnSuccess","PC_1",null,1651144298850]
+["createAnswer","PC_1",{"offerToReceiveAudio":true,"offerToReceiveVideo":true},1651144298852]
+["onsignalingstatechange","PC_0","have-remote-offer",1651144298853]
+["setRemoteDescriptionOnSuccess","PC_0",null,1651144298853]
+["createAnswer","PC_0",{"offerToReceiveAudio":true,"offerToReceiveVideo":true},1651144298854]
+["createAnswerOnSuccess","PC_1","",1651144298855]
+["setLocalDescription","PC_1","",1651144298856]
+["createAnswerOnSuccess","PC_0","",1651144298858]
+["setLocalDescription","PC_0","",1651144298859]
+["onsignalingstatechange","PC_1","stable",1651144298859]
+["setLocalDescriptionOnSuccess","PC_1",null,1651144298860]
+["onsignalingstatechange","PC_0","stable",1651144298866]
+["setLocalDescriptionOnSuccess","PC_0",null,1651144298866]
+["addIceCandidate","PC_0","",1651144298872]
+["addIceCandidateOnSuccess","PC_0",null,1651144298873]
+["setRemoteDescription","PC_0","",1651144303756]
+["onsignalingstatechange","PC_0","have-remote-offer",1651144303763]
+["onaddstream","PC_0","a5fdb50d-video-2 video:7326bfe0-41ab-464a-84ff-9438fbb21bf9",1651144303763]
+["ontrack","PC_0","video:7326bfe0-41ab-464a-84ff-9438fbb21bf9 stream:a5fdb50d-video-2",1651144303763]
+["setVideoType","PC_0",{"ssrc":3155040392,"videoType":"camera"},1651144303791]
```

Comment:
I have manually set the "PC_0" in this test case as the client does not currently send the peer connection id (as you can attest in the code here https://github.com/jitsi/jitsi-meet/pull/11426/files#diff-48868bb99f6651ada5e4d5b0fb231855f9e4a60a75e4fef20575a9d33bf61270R115).

I have noticed that we store the peer connection id as a private field on the peer connection. I am tempted to access it from there but it doesn't feel very clean. @andrei-gavrilescu do you see a better way?

Reply:
I think that's fine.. at least I don't see any strong reason for it to be private.

Reply:
We could potentially keep a list of ssrcs and their associated peer connection ids on rtcstats client and pick it up from there when we call `sendVideoTypeData`. There is some potential collision risk there but we can live with that, I think?

Reply:
Agreed that either way, make the ID public, or create a dictionary of ssrc -> IDs should be fine. 
My two recommendations for the ssrc dictionary solution would be 1) to keep it only to sender SSRCs as that should minimize the risk of clashes. And 2) to attempt to detect the collision on the rtcstats client side and rather not send the videoType message in this case, rather than polluting our DB with incorrect data.

Reply:
I like @andrei-gavrilescu's suggestion because it's just so clever! The only problem is that I can't accommodate @nils-ohlmeier 1st suggestion because we are processing the receive SSRCs in this code :-( If you guys don't have any strong feelings I will try bubbling up the peer connection id and, if it becomes too horrible, we can re-evaluate.

Reply:
Oh you are saying you need to be able to lookup SSRCs for receiver tracks as well, because the receiver tracks show up in the stats entries?
While that is true, I expect that the new `setVideoType` events would only show up on the client which sends a track. Does the event get propagated to the receivers as well?
Sorry for asking stupid questions here. I'm probably just missing some important parts in my mental model here.

Reply:
Had an offline discussion with @nils-ohlmeier yesterday where we clarified a few things and we agreed to pursue @andrei-gavrilescu idea. I will follow-up with a commit soon